### DISCUSSION TITLE
This MathJax code is working in the mathjax demo but not working in GitHub. Why?

### DISCUSSION THREAD

**Discussion by SageofPHY6Paths** [community/community#17294] [Other Feature Feedback, Questions, & Ideas]
### Issue Summary

The following code is working perfectly fine in the [mathjax demo](https://www.mathjax.org/#demo), math and physics stack exchange but not working in Github even though it is [using the same MathJax][1]. What is the problem here?

`$$\boxed{ \begin{aligned} \mathcal{L}=&-\frac{1}{4} B_{\mu \nu} B^{\mu \nu}-\frac{1}{8} t r\left(\mathbf{W}_{\mu \nu} \mathbf{W}^{\mu \nu}\right)-\frac{1}{2} t r\left(\mathbf{G}_{\mu \nu} \mathbf{G}^{\mu \nu}\right) &{\text{(U(1), SU(2) and SU(3) gauge terms)}}\\ &+\left(\bar{\nu}_{L}, \bar{e}_{L}\right) \tilde{\sigma}^{\mu} i D_{\mu}\left(\begin{array}{c} \nu_{L} \\ e_{L} \end{array}\right)+\bar{e}_{R} \sigma^{\mu} i D_{\mu} e_{R}+\bar{\nu}_{R} \sigma^{\mu} i D_{\mu} \nu_{R}+(\text { h.c. }) &{\text{(lepton dynamical term)}}\\ &-\frac{\sqrt{2}}{v}\left[\left(\bar{\nu}_{L}, \bar{e}_{L}\right) \phi M^{e} e_{R}+\bar{e}_{R} \bar{M}^{e} \bar{\phi}\left(\begin{array}{c} \nu_{L} \\ e_{L} \end{array}\right)\right] &{\text{(electron, muon, tauon mass term)}}\\ &-\frac{\sqrt{2}}{v}\left[\left(-\bar{e}_{L}, \bar{\nu}_{L}\right) \phi^{*} M^{\nu} \nu_{R}+\bar{\nu}_{R} \bar{M}^{\nu} \phi^{T}\left(\begin{array}{c} - e_{L} \\ \nu_{L} \end{array}\right)\right] &{\text{(neutrino mass term)}}\\ &+\left(\bar{u}_{L}, \bar{d}_{L}\right) \tilde{\sigma}^{\mu} i D_{\mu}\left(\begin{array}{c} u_{L} \\ d_{L} \end{array}\right)+\bar{u}_{R} \sigma^{\mu} i D_{\mu} u_{R}+\bar{d}_{R} \sigma^{\mu} i D_{\mu} d_{R}+(\text { h.c. })&{\text{(quark dynamical term)}} \\ &-\frac{\sqrt{2}}{v}\left[\left(\bar{u}_{L}, \bar{d}_{L}\right) \phi M^{d} d_{R}+\bar{d}_{R} \bar{M}^{d} \bar{\phi}\left(\begin{array}{c} u_{L} \\ d_{L} \end{array}\right)\right] &\text{(down, strange, bottom mass term)}\\ &-\frac{\sqrt{2}}{v}\left[\left(-\bar{d}_{L}, \bar{u}_{L}\right) \phi^{*} M^{u} u_{R}+\bar{u}_{R} \bar{M}^{u} \phi^{T}\left(\begin{array}{c} -d_{L} \\ u_{L} \end{array}\right)\right] &{\text{(up, charmed, top mass term)}}\\ &+\overline{\left(D_{\mu} \phi\right)} D^{\mu} \phi-m_{h}^{2}\left[\bar{\phi} \phi-v^{2} / 2\right]^{2} / 2 v^{2} &\text{(Higgs dynamical and mass term)} \end{aligned} }$$`

  [1]: https://github.blog/changelog/2022-05-19-render-mathematical-expressions-in-markdown/

### Steps to Reproduce:

1. Type the given code in GitHub markdown and see. Simpler codes work properly for example:

$$\boxed{G_{\mu\nu}+\Lambda g_{\mu\nu}=\dfrac{8\pi G}{c^4}T_{\mu\nu}}$$

**Comment by fgnievinski** [comment]
another case:

code: `blah $d_i^q$ blah $\delta d_i^q$ blah`

rendering: blah $d_i^q$ blah $\delta d_i^q$ blah
